src/go/rpk/pkg/cli/debug/bundle/bundle.go

```diff
@@ -162,7 +162,7 @@ func NewCommand(fs afero.Fs, p *config.Params) *cobra.Command {
 	f.StringVar(&logsSince, "logs-since", "", "Include log entries on or newer than the specified date (journalctl date format, e.g. YYYY-MM-DD")
 	f.StringVar(&logsUntil, "logs-until", "", "Include log entries on or older than the specified date (journalctl date format, e.g. YYYY-MM-DD")
 	f.StringVar(&logsSizeLimit, "logs-size-limit", "100MiB", "Read the logs until the given size is reached (e.g. 3MB, 1GiB)")
-	f.StringVar(&controllerLogsSizeLimit, "controller-logs-size-limit", "20MB", "The size limit of the controller logs that can be stored in the bundle (e.g. 3MB, 1GiB)")
+	f.StringVar(&controllerLogsSizeLimit, "controller-logs-size-limit", "130MB", "The size limit of the controller logs that can be stored in the bundle (e.g. 3MB, 1GiB)")
```

Comment:
?

Reply:
When doing the logic of whether to include the controller log in the debug bundle rpk also looks at the logical file size. Given that on most systems the initial segment falloc/truncate step is 32MB the controller log would basically never get included in the debug bundle anymore. Hence the need for the increase.

I think changing it to 130MB such that we at least include a single segment in case the controller log is large (128MiB is the default segment size) is good either way.

I do realize now though that that is actually in MB and not MiB so I will need to change it to 132MB.